source/epidemiology-other-compartmental-models.rst

```diff
@@ -133,7 +135,9 @@ Once again, we can see the differential equation solutions as the average of the
   :width: 600px
   :figclass: align-center
 
-  $\lambda_S$=0.6/week, $\lambda_A$=0.25/week, $\lambda_I$=0.25/week, $\gamma$=0.6
+  $\lambda_S$=1.2/14 per day, $\lambda_A$=1/28 per day, $\lambda_I$=1/28 per day, $\gamma$=0.6, $N$=10,000
```

Comment:
`1.2/14` looks clunky. In general, try to either get it as decimals (0.4, 0.3, 0.08, etc.) or as more or less simple fractions (1/4, 1/7, 1/10, 1/14, etc.). Also, perhaps `(day)^{-1}` would be a better unit. Fine for now, though, I'm merging this.